src/modules/core/components/password-requirements/password-requirements.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { type FC, useEffect, useRef } from 'react';
+import { useBoolean } from 'usehooks-ts';
+import clsx from 'clsx';
+// components
+import { Typography } from '@/modules/core/components';
+// validation
+import {
+  uppercaseCharacterRegex,
+  lowercaseCharacterRegex,
+  digitRegex,
+  passwordMinLength,
+} from '@/modules/auth/validation/schemas/sign-up-validation.schemas';
```

Comment:
I suppose you have created this component so we could apply it to forgot or change password. In that case I think it would be good not to use connection to sign-up schema.